app/views/complaints/_complaints_table_detail.html.erb

```diff
@@ -0,0 +1,22 @@
+<tr>
+  <% formatted_status = status(complaint[:attributes]) %>
+  <th scope="row" role="rowheader">
+    <span class="<%= formatted_status == "New" ? "text-bold" : "" %>">
+      <%= complaint[:id] %>
+    </span>
+  </th>
+  <td 
+    data-sort-value="<%= DateTime.parse(complaint[:attributes][:creationDate]).to_i %>"
```

Comment:
[suggestion] I might just be too helper-happy, but these long chains in views start feeling like they could be pulled out into helper methods. Could be useful on the `show` page, also, or if Date Updated gets added to this table.  Not necessary for now, just a thought.